test/parallel/test-fs-write.js

```diff
@@ -70,3 +70,14 @@ fs.open(fn2, constants.O_CREAT | constants.O_WRONLY | constants.O_TRUNC, 0o644,
             assert.strictEqual(expected, found);
           }));
         }));
+
+
+fs.open(fn, 'w', 0o644, common.mustCall(function(err, fd) {
+  assert.ifError(err);
+  console.log('open done');
```

Comment:
I think these `console` statements can be left out.

Reply:
done, thanks!